app/lib/features/segment_deletion_config.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module Features
+  class SegmentDeletionConfig
+    def self.configure(config = {})
+      @configuration = new(config)
+    end
+
+    def self.configuration
+      (@configuration || configure)
+    end
+
+    def self.config
+      configuration.config
```

Comment:
this is an instance variable on class level? 🇫🇷

Reply:
@macejmic Yes 😂 It was originally a module because we only need one global config for a feature, but @hallelujah explained me that it is better to have it like a class, the variable `@configuration` of the class (which is one of the instances) is the global configuration for the feature, and we can have as many instances as we want with different configs to test them in isolation without having to do dirty stuff from the tests like https://github.com/3scale/porta/pull/589/files#diff-02fd35c4c56ad40080442d610f0cc910L7

Reply:
ok, that could work :)